test/main.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
The `test/main.go` seems the same as `example/structured-contents/main.go`.
Do we need this?